examples/sdk-usage/README.md

```diff
@@ -0,0 +1,13 @@
+# SDK Usage Examples
+
+This is a simple example that demonstrates how to use and configure the OpenTelemetry SDK. 
+
+## Prerequisites
+* Java 1.8 or higher
+
+
+## Compile
+Compile with 
+```shell script
+./gradlew fatJar
```

Comment:
will the jar end up being runnable with `java -jar ...` ?

Reply:
You right, I should add the cmd line to run the examples.